recipes/ckanweb-deploy.rb

```diff
@@ -322,3 +336,10 @@
 	group "root"
 	mode "0755"
 end
+
+file "/etc/cron.hourly/ckan-harvest-run" do
+	content "/usr/local/bin/pick-job-server.sh && echo '{}' | #{paster} --plugin=ckanext-harvest harvester run -c #{config_file} 2>&1 > /dev/null\n"
```

Comment:
Do we need to use `pick-job-server.sh` here, or is it ok for each server to run it?

Reply:
I added `pick-job-server.sh` based on your comment in the DQL2-13 story: https://salsadigital.atlassian.net/browse/DQL2-13?focusedCommentId=134766 -- should I remove it?

Reply:
If the job can run simultaneously on multiple servers without conflict and without doubling up, then I'd like to let it do that. Picking a single server is really only for cases where race conditions are a problem.